gamestonk_terminal/cryptocurrency/coinpaprika/coinpaprika_view.py

```diff
@@ -0,0 +1,1541 @@
+            return
+
+        df = paprika.get_coins_market_info(quotes=ns_parser.vs).sort_values(
+            by=ns_parser.sortby, ascending=ns_parser.descend
+        )
+
+        if df.empty:
+            return
+
+        df = df.applymap(long_number_format_with_type_check)
+        print("")
+        print(f"Displaying data vs {ns_parser.vs}")
+        print(
+            tabulate(
+                df.head(ns_parser.top),
+                headers=df.columns,
+                floatfmt=".3f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def all_coins_info(other_args: List[str]):
+    """Displays basic coin information for all coins from CoinPaprika API
+
+    Parameters
+    ----------
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="info",
+        add_help=False,
+        description="""Show basic coin information for all coins from CoinPaprika API
+        You can display only top N number of coins with --top parameter.
+        You can sort data by rank, name, symbol, price, volume_24h, circulating_supply, total_supply, max_supply,
+        market_cap, beta_value, ath_price --sort parameter and also with --descend flag to sort descending.
+        Displays:
+            rank, name, symbol, price, volume_24h, circulating_supply,
+            total_supply, max_supply, market_cap, beta_value, ath_price
+        """,
+    )
+
+    parser.add_argument(
+        "--vs",
+        help="Quoted currency. Default USD",
+        dest="vs",
+        default="USD",
+        type=str,
+        choices=CURRENCIES,
+    )
+
+    parser.add_argument(
+        "-t", "--top", default=20, dest="top", help="Limit of records", type=int
+    )
+
+    parser.add_argument(
+        "-s",
+        "--sort",
+        dest="sortby",
+        type=str,
+        help="Sort by given column. Default: rank",
+        default="rank",
+        choices=[
+            "rank",
+            "name",
+            "symbol",
+            "price",
+            "volume_24h",
+            "circulating_supply",
+            "total_supply",
+            "max_supply",
+            "ath_price",
+            "market_cap",
+            "beta_value",
+        ],
+    )
+
+    parser.add_argument(
+        "--descend",
+        action="store_false",
+        help="Flag to sort in descending order (lowest first)",
+        dest="descend",
+        default=True,
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = paprika.get_coins_info(quotes=ns_parser.vs).sort_values(
+            by=ns_parser.sortby, ascending=ns_parser.descend
+        )
+
+        if df.empty:
+            return
+
+        df = df.applymap(lambda x: long_number_format_with_type_check(x))
```

Comment:
We probably can slice the dataframe to not have this applied to "Rank", as it would lead to 2.4K, which isn't great since this is actually 2400. If it is 2401 we have 2.401K which adds 2 characters and makes it larger.

This comment can be extended for all other applymaps that are used on rankings